chapters/en/chapter7/5.mdx

```diff
@@ -727,18 +728,41 @@ model.fit(
 )
 ```
 
-We got some loss values during training, but really we'd like to see the ROUGE metrics we computed earlier. To get those metrics, we'll need to generate outputs from the model and convert them to strings. Let's build some lists of labels and predictions for the ROUGE metric to compare (note that if you get import errors for this section, you may need to`!pip install tqdm`):
+We got some loss values during training, but really we'd like to see the ROUGE metrics we computed earlier. To get those metrics, we'll need to generate outputs from the model and convert them to strings. Let's build some lists of labels and predictions for the ROUGE metric to compare (note that if you get import errors for this section, you may need to`!pip install tqdm`). We're also going to use a trick that hugely increases performance here - [compiling our generation code with XLA](https://huggingface.co/blog/tf-xla-generate). This works a lot better when our input shapes don't vary too much, so we pad them to a fixed size, and make a new dataset with that padding collator:
```

Comment:
If you agree with my previous suggestion, we could copy-paste it here too

Reply:
Done!